lib/jsdom/living/postmessage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const isValidTargetOrigin = require("../utils").isValidTargetOrigin;
+
+module.exports = function (message, targetOrigin) {
+  let window = this;
+  let event;
+
+  if (!targetOrigin) {
+    throw new TypeError("'postMessage' requires a second argument 'targetOrigin'");
+  }
+
+  if (!isValidTargetOrigin(targetOrigin)) {
+    throw new SyntaxError("Failed to execute 'postMessage' on 'Window': " +
+      "Invalid target origin '" + targetOrigin + "' in a call to 'postMessage'.");
+  }
+
+  if (targetOrigin !== "*" && targetOrigin !== window.origin) {
```

Comment:
Just to clarify, since I can't find a ton more information on the `/` option, using a `/` as `targetOrigin` means that it only proceeds if `sendingWindow.origin === targetWindow.origin`?

> If the targetOrigin argument is a single literal U+002F SOLIDUS character (/), and the Document of the Window object on which the method was invoked does not have the same origin as the responsible document specified by the entry settings object, then abort these steps silently.